### ISSUE TITLE
Network errors

### ISSUE THREAD

**Issue by tmattio** [emillon/opam-repository#4]
## Bad Checksums

- [ ] `elina.1.3 (failed: Failed to get sources of elina.1.3: Bad checksum)`
- [ ] `ezjs_fetch.0.1 (failed: Failed to get sources of ezjs_fetch.0.1: Bad checksum)`
- [ ] `http-cookie.3.0.0 (failed: Failed to get sources of http-cookie.3.0.0: Bad checksum)`
- [x] ocaml-ffmpeg (https://github.com/savonet/ocaml-ffmpeg/archive/v1.0.0-rc1.tar.gz):
  - [x] ffmpeg-av.1.0.0~rc1
  - [x] ffmpeg-avcodec.1.0.0~rc1
  - [x] ffmpeg-avdevice.1.0.0~rc1
  - [x] ffmpeg-swresample.1.0.0~rc1
  - [x] ffmpeg-swscale.1.0.0~rc1
  - [x] ffmpeg.1.0.0~rc1
  - [x] ffmpeg-avfilter.1.0.0~rc1

## Curl error 404

- [ ] `proverif.1.97pl3 (failed: Failed to get sources of proverif.1.97pl3: curl error code 404)`
- [ ] `proverif.1.98 (failed: Failed to get sources of proverif.1.98: curl error code 404)`
- [ ] `proverif.1.98pl1 (failed: Failed to get sources of proverif.1.98pl1: curl error code 404)`

## Curl error 403

- [ ] `javalib.2.3.5 (failed: Failed to get sources of javalib.2.3.5: curl error code 403)`
- [ ] `sawja.1.5.4 (failed: Failed to get sources of javalib.2.3.5: curl error code 403)`

**Comment by emillon** [comment]
Thanks for the list. Can you remove the duplicates?

**Comment by tmattio** [comment]
Done :)

**Comment by Leonidas-from-XIV** [comment]
The only tarball I could find in the cache was ffmpeg, so I made a PR to import it: https://github.com/ocaml/opam-source-archives/pull/16. Once this is merged we can update the OPAM file.

**Comment by tmattio** [comment]
What is the fix for the packages for which we can't find the tarball? Should we make them unavailable on opam?

**Comment by Leonidas-from-XIV** [comment]
@emillon mentioned there might be a way to reconstruct the original tarball from a new github generated one, but I am not sure how.

Anyway, if we can't find any tarball at all the probably need to be marked unavailable since it is impossible to build them; if we find a different tarball we could reach out to the authors and ask whether these are equivalent and update the checksums (and probably make sure the tarball stays available).

I feel like OPAM could use a package mirror (possibly with some kind of CDN in front of it), but that's probably out of scope for now.
